regression/contracts/assigns_enforce_15/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=10$
+^SIGNAL=0$
+^\[bar.1\] line 13 .*: FAILURE$
+^\[baz.1\] line 18 .*: FAILURE$
```

Comment:
This looks like it will fail, isn't it line 19? I think other tests use regex like:
```
^\[main.enum-range-check.6\] line \d+ enum range check in ev3: SUCCESS$
^\[main.enum-range-check.7\] line \d+ enum range check in ev2: FAILURE$
```
taken from `regression/cbmc/enum_is_in_range/enum_test5.desc`

Reply:
Thanks, @TGWDB! Implemented as suggested.